src/main/kotlin/app/CacheService.kt

```diff
@@ -0,0 +1,101 @@
+@file:Suppress("SqlResolve")
+
+package app
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.slf4j.LoggerFactory
+import java.sql.DriverManager
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Date
+
+object CacheService {
+    private const val urlToDb = "jdbc:h2:mem:userinfo"
+    private val log = LoggerFactory.getLogger(CacheService.javaClass)
+    private val objectMapper = jacksonObjectMapper()
+        .registerModule(SimpleModule().addDeserializer(Date::class.java, DateDeserializer()))
```

Comment:
perhaps we can use the jdk 8 module to deserialize directly to `Instant`'s as well

Reply:
Indeed. It just works out of the box!